tests/end_to_end/test_basic_cases/test_ip_reputation/test_ip_reputation.py

```diff
@@ -59,6 +60,9 @@
 
 configurations, configuration_metadata, cases_ids = config.get_test_cases_data(test_cases_file_path)
 
+# Marks
+pytestmark = [TIER0, LINUX]
```

Comment:
This test needs a Windows agent, too.

Reply:
Done in d5687d0bf34e45289c3be8be4bb4e3382571a2e2